tsfeatures/tsfeatures.py

```diff
@@ -218,12 +237,11 @@ def flat_spots(x: np.array, freq: int = 1) -> Dict[str, float]:
     try:
         cutx = pd.cut(x, bins=10, include_lowest=True, labels=False) + 1
     except:
-        return {'flat_spots': np.nan}
-
-    rlex = np.array([sum(1 for i in g) for k,g in groupby(cutx)]).max()
+        return {"flat_spots": np.nan}
 
-    return {'flat_spots': rlex}
+    rlex = np.array([sum(1 for i in g) for k, g in groupby(cutx)]).max()
```

Comment:
I think there's a missing return here

Reply:
add a dict with the rlex as return value